pkg/controllers/kube-controller-manager.go

```diff
@@ -171,7 +173,11 @@ func GetKubeControllerManagerArgs(config map[string]interface{}) []string {
 	args := []string{}
 	for key, value := range extendedArguments.(map[string]interface{}) {
 		for _, arrayValue := range value.([]interface{}) {
-			args = append(args, fmt.Sprintf("--%s=%s", key, arrayValue.(string)))
+			if len(key) == 1 {
+				args = append(args, fmt.Sprintf("-%s=%s", key, arrayValue.(string)))
```

Comment:
This should work with [double dash](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/#options) too.

Reply:
yes, you are right. But for example `-h` doesn't, so it shouldn't hurt anything to keep this here.

Reply:
also a minor plus for consistency is that we are using `-v` in KCM operator as well when generating the args.

Nevertheless, I don't have a strong preference if you want to remove this.

Reply:
log level verbosity flag in KCM supports simple or double dash. So this sounds good to me.